src/FluentMigrator.Runner.Oracle/Processors/DotConnectOracle/DotConnectOracleProcessor.cs

```diff
@@ -208,7 +209,24 @@ protected override void Process(string sql)
             EnsureConnectionIsOpen();
 
             using (var command = CreateCommand(sql))
-                command.ExecuteNonQuery();
+            {
+                try
+                {
+                    command.ExecuteNonQuery();
+                }
+                catch (Exception ex)
+                {
+                    using (var message = new StringWriter())
+                    {
+                        message.WriteLine("An error occured executing the following sql:");
```

Comment:
This looks great.  Can you consider consolidating the code across the three Processors so that there is a standard way to catch these exceptions and log the error?  Even if SQL Server and Postgres are a bit different, I think a SqlExceptionUtility class would be useful to avoid repeating logic everywhere.

Reply:
This is what I have in mind:

```c#
public static class SqlExceptionUtility
{
  public static void Throw(string sql, Exception ex)
  {
    using (var message = new StringWriter())
    {
       message.WriteLine("An error occured executing the following sql:");
       message.WriteLine(sql);
       message.WriteLine("The error was {0}", ex.Message);
       throw new Exception(message.ToString(), ex);
    }
  }
}
```

Reply:
It should go in the Runner.csproj

Reply:
@jzabroski , I added it as a virtual method to ProcessorBase.cs.
This seemed closer to the classes that would utilize it.